unit-tests/src/test/java/org/eclipse/collections/impl/block/procedure/ToStringTest.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2021 The Bank of New York Mellon.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * and Eclipse Distribution License v. 1.0 which accompany this distribution.
+ * The Eclipse Public License is available at http://www.eclipse.org/legal/epl-v10.html
+ * and the Eclipse Distribution License is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ */
+
+package org.eclipse.collections.impl.block.procedure;
+
+import org.eclipse.collections.api.factory.Lists;
+import org.eclipse.collections.impl.test.Verify;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class ToStringTest
```

Comment:
This should not be a test. There should be methods on `ChainedProcedureTest` and `CollectionRemoveProcedureTest` for toString.

Reply:
Fixed.